### PR TITLE
add DS-License.md

### PR DESCRIPTION
**Pull request by CalvinKirs**
add DS-License.md

### CODE REVIEW

File: docs/zh-cn/development/DS-License.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+注：本文仅使用于Apache项目。
+
+### Apache项目可接受的License
+
+当您想要为DolphinScheduler（亦或其他Apache项目）增添一个新的功能，这个功能涉及到其他开源软件的引用，那么您必须注意，目前Apache项目支持遵从以下协议的开源软件（如果有遗漏，欢迎补充）：
+
+[ASF第三方许可证策](https://apache.org/legal/resolved.html)
+
+如果您所使用的第三方软件并不在以上协议之中，那么很抱歉，您的代码将无法通过审核，建议您找寻其他替代方案。
```
Comment (dailidong) [inline docs/zh-cn/development/DS-License.md:14]: 软件 -> 软件(包含但不限于第三方的jar、文本、css、js、图片、图标、音视频等及在第三方基础上做的修改)

File: docs/zh-cn/development/DS-License.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+另外，当您需要使用新的软件的时候，请将您这样做的原因、最终产出结果发邮件至dev@dolphinscheduler.apache.org讨论，当得到至少3票PPMC认同的时候，您方可以引入。
+
+### 如何在DolphinScheduler合法的使用第三方开源软件
+
+当我们想要引入一个新的第三方软件至我们的项目中的时候，除了他们所遵从的协议是Apache允许的，另外一点很重要，就是合法的使用。您可以参考以下文章
+
+* [COMMUNITY-LED DEVELOPMENT "THE APACHE WAY"](https://apache.org/dev/licensing-howto.html)
+
+
+以Apache为例，当我们使用了zookeeper，那么我们必须申明，我使用了当我们使用了zookeeper，那么关于当我们使用了zookeeper的NOTICE文件（每个开源项目都会有NOTICE文件，一般位于根目录），则必须在我们的项目中体现，用Apache的话来讲，就是"Work" shall mean the work of authorship, whether in Source or Object form, made available under the License, as indicated by a
```
Comment (dailidong) [inline docs/zh-cn/development/DS-License.md:25]: change zookeeper to ZooKeeper

File: docs/zh-cn/development/DS-License.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+copyright notice that is included in or attached to the work.
+
+关于具体的各个开源协议使用协议，在此不做过多篇幅一一介绍，有兴趣可以自行查询了解。
+
+### DS-License 检测规则
```
Comment (dailidong) [inline docs/zh-cn/development/DS-License.md:30]: DS -> DolphinScheduler

**Comment by dailidong** [comment]
good job

**Comment by CalvinKirs** [comment]
@dailidong Thank you very much for your review, I have completed the changes, please check again

**Pull request merged**